guides/common/assembly_configuring-puppet.adoc

```diff
@@ -0,0 +1,27 @@
+ifdef::context[:parent-context-of-assembly_configuring-puppet: {context}]
+
+ifndef::context[]
+[id="assembly_configuring-puppet"]
+endif::[]
+ifdef::context[]
+[id="assembly_configuring-puppet_{context}"]
+endif::[]
+= Configuring Puppet
+
+:context: assembly_configuring-puppet
```

Comment:
Why do you want to set the context here? I don't yet see the point or even if this works as expected.

Can we go with the following:

```
[id="Configuring_Puppet_{context}"]
= Configuring Puppet
```

Reply:
I think that this context is set for the modules included inside the assembly. IMHO the context changes depending on how the files are nested by inclusions, i.e. an assembly should have the context of the parent guide, whereas included modules should have the context of the parent assembly.
But I agree, we need to test it. I'll test if it works with make build.
This is again suggested by Red Hat templates for [docs modularization](https://redhat-documentation.github.io/modular-docs/#assembly-guidelines) and we're going to need it for our Satellite publishing.
So, no, I'm sorry, but with clear conscience, we can't go with your suggestion.

Reply:
The context is being set for the guide in the master.adoc for each guide it is referenced in. 
We've never had problems building before this, so I tend to also question the need for this.